web/app/themes/radcliffe/header.php

```diff
@@ -2,46 +2,49 @@
 
 <html <?php language_attributes(); ?>>
 
-<head profile="http://gmpg.org/xfn/11">
+<head>
 
     <meta http-equiv="Content-Type" content="<?php bloginfo('html_type'); ?>; charset=<?php bloginfo('charset'); ?>"/>
-    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=yes">
 
     <?php wp_head(); ?>
 
 </head>
 
 <body <?php body_class(); ?>>
 
+<nav aria-label="Skip Links" class="off-screen"><a href="#maincontent" class="maincontent skip-link">Skip to main content</a></nav>
```

Comment:
Please remove the 'off-screen' class - testing has shown that the element remains hidden when tabbing

Reply:
Ok.